Sources/ArcGISToolkit/Components/OfflineMapAreasView/PreplannedMapModel.swift

```diff
@@ -16,22 +16,74 @@ import SwiftUI
 import ArcGIS
 
 /// An object that encapsulates state about a preplanned map.
-@MainActor
```

Comment:
Do you want to manually determine the actor context of each method and property?

If it is for the `Hashable` extension, we can instead mark those protocol requirements `nonisolated`.